src/test/java/com/vfasad/test/service/OrderServiceTest.java

```diff
@@ -0,0 +1,312 @@
+package com.vfasad.test.service;
+
+import com.vfasad.entity.*;
+import com.vfasad.exception.NotFoundException;
+import com.vfasad.repo.OrderRepository;
+import com.vfasad.service.EmailService;
+import com.vfasad.service.OrderService;
+import com.vfasad.service.ProductService;
+import com.vfasad.service.TeamService;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.data.domain.Sort;
+
+import java.math.BigInteger;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.*;
+
+import static org.mockito.Mockito.*;
+import static org.junit.Assert.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OrderServiceTest {
+    private static final Long ORDER_ID = 1L;
+    private static final double AREA = 2.45;
+    private static final int CLIP_COUNT = 5;
+    private static final int FURNITURE_SMALL_COUNT = 8;
+    private static final int FURNITURE_BIG_COUNT = 2;
+    private static final String DOCUMENT = "document";
+    private static final double PRICE = 4566.78;
+    private static final Long USER_ID = 1L;
+    private static final Long TEAM_ID = 1L;
+    private static final long[] CONSUME_IDS = new long[] {1L, 2L, 3L};
```

Comment:
Short form could be used (just a bit more readable):
`private static final long[] CONSUME_IDS = {1L, 2L, 3L};`